packages/api-explorer/src/test-utils/render_with_search.tsx

```diff
@@ -34,7 +62,7 @@ export const renderWithSearchAndRouter = (
   options?: Omit<RenderOptions, 'queries'>
 ) => {
   const RoutedComponent = withRouter(component, initialRouterEntries)
-  return render(
+  return renderWithTheme(
     withSearchProvider(RoutedComponent, { pattern, criteria }),
     options
```

Comment:
I like this change! Unfortunately there are places where we were doing
```
renderWithSearch(withThemeProvider(...))
// or
renderWithRouter(withThemeProvider(...))
// or
renderWithSearchAndRouter(withThemeProvider(...))
```
which will require cleaning up. They probably still work since the component is just being wrapped with the provider twice but it might be a good idea to do that cleanup.

Reply:
Those are all cleaned up now